src/pages/Explore/LogExplorationPage.tsx

```diff
@@ -1,15 +1,16 @@
 import React, { useEffect, useState } from 'react';
-import { newLogsExploration } from '../../utils/utils';
-import { LogExploration } from './LogExploration';
+
 import { getUrlSyncManager } from '@grafana/scenes';
+import { MainComponent } from '../../components/Main/MainComponent';
+import { newLogsExploration } from 'utils/utils';
 
 export const LogExplorationPage = () => {
   const [exploration] = useState(newLogsExploration());
 
   return <LogExplorationView exploration={exploration} />;
 };
 
-export function LogExplorationView({ exploration }: { exploration: LogExploration }) {
```

Comment:
I would rename this ExploreLogs, because it's not Explore.

Reply:
What do you mean by "this"? The route is called `explore` I think that's why the page is called like that.

Reply:
If, for some reason, we move the link from Explore / Logs to, I don't know, App / Logs, should we refactor the app?

I think it's mixing concerns.